tests/test_operation2.py

```diff
@@ -706,7 +706,7 @@ def test_oauth_scopes_in_or(security_handler_factory):
 
 
 def test_form_transformation(api):
-    mock_self = mock.Mock(strict_validation=True)
+    mock_self = mock.Mock()
```

Comment:
`strict_validation` for form data is checked by the `FormDataValidator`, we don't need to include it in the transformation here.